Allow typing_extensions.Protocol and typing.Protocol to mix (#237)
python/typing_extensions: doc/index.rst

      Backported changes to runtime-checkable protocols from Python 3.12,
      including :pr-cpy:`103034` and :pr-cpy:`26067`.

   .. versionchanged:: 4.7.0

>       Classes can now have both :py:class:`typing.Protocol` and
>       ``typing_extensions.Protocol`` in their mro. Previously, this led to

Reviewer: ```suggestion
      Classes can now inherit from both :py:class:`typing.Protocol` and
      ``typing_extensions.Protocol`` simultaneously. Previously, this led to
```
